CONTRIBUTORS.txt

```diff
@@ -1,3 +1,9 @@
+# This file is autocompleted by 'contributors-txt',
+# using the configuration in '/Users/daniel/DocumentenLaptop/Programming/Github/astroid/script/.contributors_aliases.json'.
+# Do not add new persons manually and only add information without
+# using '-' as the line first character.
+# Please verify that your change are stable if you modify manually.
+
```

Comment:
```suggestion
```
Apparently the header only 'worked on my machine' :smile:

Reply:
What's bad about this warning?

Reply:
It's duplicated it should not have been added by contributor-txt. (see 4 lines under)